x/fswap/types/keys.go

```diff
@@ -17,6 +17,12 @@ const (
 	MemStoreKey = "mem_fswap"
 )
 
+var (
+	ParamsKey                 = byte(0x00)
+	SwappedKey                = byte(0x01)
+	SwappableNewCoinAmountKey = byte(0x02)
```

Comment:
Can we prune it?

Reply:
Sorry, I forget to delete `SwappableNewCoinAmount`, `GetSwappableNewCoinAmount` &`SetSwappableNewCoinAmount`